model/src/pyrenew/latent/infections.py

```diff
@@ -48,24 +48,15 @@ class Infections(RandomVariable):
 
     def __init__(
         self,
-        infections_mean_varname: str = "latent_infections",
     ) -> None:
         """
         Default constructor for Infections class.
 
-        Parameters
-        ----------
-        infections_mean_varname : str, optional
-            Name to be assigned to the deterministic variable in the model.
-            Defaults to "latent_infections".
-
         Returns
         -------
         None
         """
```

Comment:
Any further plans for the `Infections` constructor? The removal of naming variable made me think _looks sparse, are there things we are missing?_.

Reply:
Good catch. I will remove it.